src/main/flight/mixer_init.c

```diff
@@ -343,12 +343,11 @@ void mixerInitProfile(void)
     }
     mixerRuntime.dynIdlePGain = currentPidProfile->dyn_idle_p_gain * 0.00015f;
     mixerRuntime.dynIdleIGain = currentPidProfile->dyn_idle_i_gain * 0.01f * pidGetDT();
-    mixerRuntime.dynIdleDGain = currentPidProfile->dyn_idle_d_gain * 0.0000003f * pidGetPidFrequency();
+    mixerRuntime.dynIdleDGain = currentPidProfile->dyn_idle_d_gain * 0.0000003f * pidGetPidFrequency() * 800 * pidGetDT() / 20.0f;
```

Comment:
in pure math world pidGetPidFrequency() *  pidGetDT() will be equal to 1, does it make sense to multiply them here?